app/src/main/java/com/babylon/wallet/android/presentation/transaction/fees/TransactionFees.kt

```diff
@@ -123,13 +123,7 @@ data class TransactionFees(
      * Finalized fee to lock for the transaction
      **/
     val transactionFeeToLock: Decimal192
-        get() = totalExecutionCost +
-            networkFinalization +
-            effectiveTip +
-            networkStorage +
-            feePaddingAmountForCalculation +
-            royalties -
-            nonContingentFeeLock
+        get() = (networkFee - nonContingentFeeLock).clamped
```

Comment:
`networkFee` in line 49

Reply:
Is this the value that will be added in the manifest?

Reply:
yes

Reply:
I don't think is correct, the Wallet should lock the max fee that includes the nonContingentFeeLock

Reply:
as we discussed in the SU, this is correct

Reply:
yup